package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tcomb-builder",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Just checking – this is considered a bug fix ya?

Reply:
yep!